### ISSUE TITLE
能提供一下导出onnx的代码吗

### ISSUE THREAD

**Issue by deep-practice** [xiaoxiaotao/person-detection#1]

**Comment by deep-practice** [comment]
我导出来的onnx和你的不太一样，想知道在concate的时候，你是怎么操作的
loc_pred = loc.view(loc.size(0), -1, 4).reshape(1,-1) #[1, 7759, 4]-->[1,7759x4]
conf_pred = self.softmax(conf.view(-1, self.num_classes)).reshape(1,-1) #[7759, 2]-->[1,7559x2]
output = torch.cat((loc_pred,conf_pred),0)
@xiaoxiaotao

**Comment by xiaoxiaotao** [comment]
他的网络输出，7759×4是位置信息，7759×2是置信度，两个都reshape成一维的再进行连接

**Comment by deep-practice** [comment]
嗯嗯 我用自己导出来的也可以推断 只是可能一些细节跟你不太一样

**Comment by xiaoxiaotao** [comment]
loc = torch.cat([o.flatten(1) for o in loc], 1)
        conf = torch.cat([o.flatten(1) for o in conf], 1)
        print(loc.shape)
        print(conf.shape)
        output=torch.cat((loc,conf),1)

**Comment by deep-practice** [comment]
没有搞懂为什么要把softmax拿出来呢

**Comment by xiaoxiaotao** [comment]
当时为了了解数据的流动，全都拆开了，进一步了解运行过程
